### PR TITLE
feature: add bcs-data-manager #1631

### PR DESCRIPTION
**Pull request by kayinli**
add bcs-data-manager

### CODE REVIEW

File: bcs-services/bcs-data-manager/pkg/datajob/workload.go [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+		Index:              common.GetIndex(opts.CurrentTime, opts.Dimension),
+		Time:               primitive.NewDateTimeFromTime(opts.CurrentTime),
+		CPURequest:         cpuRequest,
+		CPUUsage:           cpuUsage,
+		CPUUsageAmount:     cpuUsed,
+		MemoryRequest:      memoryRequest,
+		MemoryUsage:        memoryUsage,
+		MemoryUsageAmount:  memoryUsed,
+		InstanceCount:      instanceCount,
+		MaxCPUUsageTime:    minuteMetrics[len(minuteMetrics)-1].MaxCPUUsageTime,
```
Comment (DeveloperJim) [inline bcs-services/bcs-data-manager/pkg/datajob/workload.go:172]: 优化写法

File: bcs-services/bcs-data-manager/pkg/datajob/workload.go [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+		Index:              common.GetIndex(opts.CurrentTime, opts.Dimension),
+		Time:               primitive.NewDateTimeFromTime(opts.CurrentTime),
+		CPURequest:         cpuRequest,
+		CPUUsage:           cpuUsage,
+		CPUUsageAmount:     cpuUsed,
+		MemoryRequest:      memoryRequest,
+		MemoryUsage:        memoryUsage,
+		MemoryUsageAmount:  memoryUsed,
+		InstanceCount:      instanceCount,
+		MaxCPUUsageTime:    hourMetrics[len(hourMetrics)-1].MaxCPUUsageTime,
```
Comment (DeveloperJim) [inline bcs-services/bcs-data-manager/pkg/datajob/workload.go:111]: 优化写法

File: bcs-services/bcs-data-manager/pkg/bcsmonitor/client.go [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+	url := fmt.Sprintf("%s%s", c.completeEndpoint, LabelsPath)
+	if queryString != "" {
+		url = fmt.Sprintf("%s?%s", url, queryString)
+	}
+	rsp, err := c.requestClient.DoRequest(url, "GET", c.defaultHeader, nil)
+	if err != nil {
+		return nil, err
+	}
+	result := &LabelResponse{}
+	err = json.Unmarshal(rsp, &result)
```
Comment (DeveloperJim) [inline bcs-services/bcs-data-manager/pkg/bcsmonitor/client.go:155]: 指针修正

**Review by DeveloperJim — CHANGES_REQUESTED** [review]

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-bcs?pullRequest=1632) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-bcs?pullRequest=1632) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-bcs?pullRequest=1632) it.</sub>

File: bcs-services/bcs-data-manager/pkg/client/pkg/client_test.go [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+	assert.NotNil(t, rsp)
+
+}
+
+func Test_GetClusterInfo(t *testing.T) {
+	client := NewDataManagerCli(&Config{
+		APIServer: "",
+		AuthToken: "",
+	})
+	rsp, err := client.GetClusterInfo(&datamanager.GetClusterInfoRequest{
```
Comment (DeveloperJim) [inline bcs-services/bcs-data-manager/pkg/client/pkg/client_test.go:41]: 测试不存在实际意义？

**Review by DeveloperJim — CHANGES_REQUESTED** [review]

**Pull request merged**